.ci.yaml

```diff
@@ -4622,3 +4622,22 @@ targets:
       tags: >
         ["devicelab", "hostonly", "mac"]
       task_name: flutter_tool_startup__macos
+
+  # This is used to test the new release scheduler. This will eventually be deleted.
+  - name: Linux docs_test_with_release_scheduler
+    recipe: flutter/flutter
+    scheduler: release
+    bringup: true
+    timeout: 60
+    properties:
+      cores: "32"
```

Comment:
Is this meant to run against a 32 core machine?

Reply:
I believe so. This is mostly a duplicate of https://github.com/flutter/flutter/blob/master/.ci.yaml#L302

Reply:
We can probably remove it here, not sure if we have a 32 core config in dart-internal.

Reply:
Yeah, we should remove it. No 32 core bots are available: https://chrome-swarming.appspot.com/botlist?c=id&c=task&c=cores&c=os&c=status&d=asc&f=pool%3Aluci.dart-internal.ci&k=pool&s=id